src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html

```diff
@@ -1,4 +1,7 @@
-<div *ngIf="friends && friends.length>0; else spinner">
+<div *ngIf="!friends">
+  <app-spinner></app-spinner>
+</div>
+<div *ngIf="friends && friends.length > 0">
```

Comment:
you can check by the next way ` *ngIf="friends?.length"`

Reply:
changed